src/woo_publications/metadata/tests/test_admin_logging.py

```diff
@@ -0,0 +1,166 @@
+from django.urls import reverse
+
+from django_webtest import WebTest
+from maykin_2fa.test import disable_admin_mfa
+
+from woo_publications.accounts.tests.factories import UserFactory
+from woo_publications.logging.constants import Events
+from woo_publications.logging.models import TimelineLogProxy
+
+from ..constants import OrganisationOrigins
+from ..models import Organisation
+from .factories import OrganisationFactory
+
+
+@disable_admin_mfa()
+class TestOrganisationAdminAuditLogging(WebTest):
+    """
+    Test that CRUD actions on organisations are audit-logged.
+
+    We have a generic implementation in woo_publications.logging for this behaviour,
+    for which the code coverage is provided through this test class.
+
+    Additionally, there's a system check to ensure audit logging is added to our admin
+    classes, which should cover the rest of the apps/models.
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.user = UserFactory.create(superuser=True)
+
+    def test_admin_create(self):
+        assert not TimelineLogProxy.objects.exists()
+        reverse_url = reverse("admin:metadata_organisation_add")
+
+        response = self.app.get(reverse_url, user=self.user)
+
+        self.assertEqual(response.status_code, 200)
+
+        form = response.forms["organisation_form"]
+        form["naam"] = "organisation name"
+
+        form.submit(name="_save")
+
+        added_item = Organisation.objects.order_by("-pk").first()
+        assert added_item is not None
+        log = TimelineLogProxy.objects.first()
+        assert log is not None
```

Comment:
same here, you can substantially simplify this

```suggestion
        log = TimelineLogProxy.objects.get()
```